### DISCUSSION TITLE
Feature request: default image for listing

### DISCUSSION THREAD

**Discussion by blacksqr** [quarto-dev/quarto-cli#470] [Feature Requests]
If I have a listing or blog page in default format, and only some entries have an image associated with them, the entries without images just have big gray spaces where the missing image would be.

It would be nice to be able to specify a default image in the listing YAML section, rather than having to specify one in each individual posting file that doesn't have its own image.

**Comment by dragonstyle** [comment]
This is a great suggestion - I'll look into this today!

**Reply by dragonstyle** [reply]
Ok, I've added (658c3ed7a546279cdc56794492c3b35e27f2b584) the ability to specify a placeholder image like:

```yaml
---
listing:
  image-placeholder: placeholder.png
---
```

A fresh release is being built now- thanks for the feedback!

**Comment by blacksqr** [comment]
Wow, thanks for all your quick responses.

**Comment by mcanouil** [comment]
Closing as completed, see <https://github.com/quarto-dev/quarto-cli/commit/658c3ed7a546279cdc56794492c3b35e27f2b584>.
